packages/orbit-components/src/PictureCard/index.js

```diff
@@ -229,7 +229,7 @@ const PictureCard = ({
     }
   }, []);
 
-  const cardID = React.useMemo(() => randomID("pictureCardID"), []);
+  const cardID = useRandomId()("pictureCardID");
```

Comment:
In order to give `useRandomId` more weight and make it clear what's going on I recommend always splitting it into two lines instead of immediately invoking.

```js
const randomId = useRandomId();
const cardID = randomId("pictureCardID");
```

Reply:
in my opinion it makes sense only if are multiple ids are needed :)

Reply:
Economically speaking, yes, but it's really uncommon to use hooks like this, they should be on their own, not be like regular functions. This usage makes me lean more towards removing the seed implementation and just using `useUID` instead.